minesweeper_model/str_input.py

```diff
@@ -0,0 +1,28 @@
+field=[]
+mine_coord=[]
+x_len,y_len=0,0
+x_mine=0
+column=''
+
+while(True):
+	colum=str(input())
+	
+	if(colum=="done"):
+		break
+	else:
+		field.append(colum)
```

Comment:
Further to my comment below, the `field` variable is not necessary, so you can remove it.